test-lib-remote-openshift.sh

```diff
@@ -65,13 +65,18 @@ function ct_os_upload_image_external_registry() {
   local image_name=${input_name##*/}
   local imagestream=${1:-$image_name:latest}
   local output_name
+  local image_name_without_register
 
   ct_os_login_external_registry
 
   output_name="${INTERNAL_DOCKER_REGISTRY}/rhscl-ci-testing/$imagestream"
 
+  # This command removes 'registry.access.redhat.com' from 'registry.access.redhat.com/rhscl/nodejs-10-rhel7'
+  # image_name_without_register is now 'rhscl/nodejs-10-rhel7'
+  # Images from build and tag section have the name 'rhscl/nodejs-10-rhel7'
+  image_name_without_register=$(cut -d '/' -f 2- <<< "$input_name")
```

Comment:
How about `${input_name#*/}` instead? Seems a bit more readable to me and would be done the same way as setting `image_name`.

Reply:
What about if `input_name` is `rhscl/nodejs-10-rhel7`? Then the input name will be `nodejs-10-rhel7`.

Reply:
I have updated to `grep` command. If `input_name` is `rhscl/nodejs-10-rhel7` or `registry.access.redhat.com/rhscl/nodejs-10-rhel7` then `image_name_without_register` is `rhscl/nodejs-10-rhel7`.